README.md

```diff
@@ -1,2 +1,134 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Git</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **fast to load**
+- **nice blue color**
+- **easy to read**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- View -->
+
+## 🚀 View <a name="View"></a> https://luisrpavanelli.github.io/luisrpavanelli-Microverse-Project.github.io/
+
+[Click here to view]: https://luisrpavanelli.github.io/luisrpavanelli-Microverse-Project.github.io/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This is the easiest app to use; all you have to do is click on the live demo to preview it or clone it to have your own copy.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Luis Renan Pavanello**
+
+- GitHub: [@luisrpavanelli](https://github.com/luisrpavanelli)
+- LinkedIn: [luisrpavanello](https://linkedin.com/in/luisrpavanello)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Changing colors when hovering over text**
+- [ ] **Add your name**
+- [ ] **Generate a random motivational phrase**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+We welcome contributions, issues, and feature requests!
+
+Please feel free to look over the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, mark it as a favorite.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you for the opportunity to learn and grow with Microverse.org. I appreciate the platform you provide to develop our skills and knowledge. Thank for my reviewers to taking the time to review my work. Your feedback and expertise are invaluable to me, and I truly appreciate your thoughtful comments and suggestions. Your input has helped me to improve my work, and I am grateful for your time and effort. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken at the moment, I got 404 error. I observed you did not have a `LICENSE` file in your root folder, perhaps you want to link it up with `MIT.md` file. 👍